code/modules/blob/blob.dm

```diff
@@ -139,10 +52,16 @@
 	else
 		icon_state = "blob_damaged"
 
+/obj/effect/blob/process(wait, times_fired)
+	regen()
+	if(times_fired % attack_freq)
```

Comment:
Yee this won't work on Aurora code. Our processor SSs don't pass any args into process(). This is also a really dumb way to do this.

Have a variable on the blob that saves the time you last attacked. Calculate time difference. Then determine if it's enough. Yadda.